sdk/translation/Azure.AI.Translation.Document/tests/GetAllDocumentsFilterTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.Translation.Document.Tests
+{
+    public partial class DocumentTranslationClientLiveTests
+    {
+        private async Task<DocumentTranslationOperation> CreateSingleTranslationJobAsync(DocumentTranslationClient client, int docsCount = 1)
+        {
+            var testDocs = CreateDummyTestDocuments(count: docsCount);
+            var sourceContainer = await CreateSourceContainerAsync(testDocs);
+
+            var targetContainer = await CreateTargetContainerAsync();
+            var input = new DocumentTranslationInput(sourceContainer, targetContainer, "fr");
+            var translationOp = await client.StartTranslationAsync(input);
+
+            return translationOp;
+        }
+
+        [RecordedTest]
+        public async Task GetAllDocumentStatusesFilterByStatusTest()
+        {
+            // create client
+            var client = GetClient();
+
+            // create translation job
+            var operation = await CreateSingleTranslationJobAsync(client, docsCount: 3);
+            await operation.WaitForCompletionAsync();
+
+            // list docs
+            var filter = new DocumentFilter(
+                statuses: new List<DocumentTranslationStatus>()
+                {
+                    DocumentTranslationStatus.Succeeded
+                }
+            );
+            var result = operation.GetAllDocumentStatuses(filter: filter);
+
+            // assert.
+            Assert.That(result.All(d => d.Status == DocumentTranslationStatus.Succeeded));
+        }
+
+        [RecordedTest]
+        public async Task GetAllDocumentStatusesFilterByIdsTest()
+        {
+            // create client
+            var client = GetClient();
+
+            // create translation job
+            var operation = await CreateSingleTranslationJobAsync(client, docsCount: 3);
+            await operation.WaitForCompletionAsync();
+            var testIds = operation.GetAllDocumentStatuses().Select(d => d.Id).ToList().GetRange(0, 1);
+
+            // list docs
+            var filter = new DocumentFilter(
+                ids: testIds
+            );
+            var result = operation.GetAllDocumentStatuses(filter: filter);
+
+            // assert
+            Assert.That(result.All(d => testIds.Contains(d.Id)));
```

Comment:
looks like `testIds` only has one id, right? will it be better to just do 
`Assert.AreEqual(testIds[0], results.Id)` or something like that

Reply:
i made it this way to make it more generic